### ISSUE TITLE
Order reporter messages

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#4160]
```
Waiting for verification...
Cleaning up challenges
Failed authorization procedure. example.com (tls-sni-01): urn:acme:error:connection :: The server could not connect to the client to verify the domain :: Failed to connect to ip:443 for TLS-SNI-01 challenge, www.example.com (tls-sni-01): urn:acme:error:connection :: The server could not connect to the client to verify the domain :: Failed to connect to ip:443 for TLS-SNI-01 challenge

IMPORTANT NOTES:
 - If you lose your account credentials, you can recover through
   e-mails sent to root@example.com.
 - The following errors were reported by the server:

   Domain: example.com
   Type:   connection
   Detail: Failed to connect to ip:443 for TLS-SNI-01
   challenge

   Domain: www.example.com
   Type:   connection
   Detail: Failed to connect to ip:443 for TLS-SNI-01
   challenge

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A record(s) for that domain
   contain(s) the right IP address. Additionally, please check that
   your computer has a publicly routable IP address and that no
   firewalls are preventing the server from communicating with the
   client. If you're using the webroot plugin, you should also verify
   that you are serving files from the webroot path you provided.
 - Your account credentials have been saved in your Certbot
   configuration directory at /etc/letsencrypt. You should make a
   secure backup of this folder now. This configuration directory will
   also contain certificates and private keys obtained by Certbot so
   making regular backups of this folder is ideal.
```
As reported in #4135, we may want to place the account credentials messages together.

For the developer that fixes this, we probably either need to change the priority of the messages or change the data structure used in `reporter.py`.

**Comment by bmw** [comment]
Actually, account recovery was removed from the [ACME spec](https://tools.ietf.org/html/draft-ietf-acme-acme-02) in version 2 and Let's Encrypt has no plans to implement it (see letsencrypt/boulder#855). We should just remove the first message under "IMPORTANT NOTES:".

**Comment by bmw** [comment]
This was fixed in #4212.
